enhancements/olm/simplify-apis.md

```diff
@@ -129,7 +129,7 @@ status:
   components:
    matchLabels:
       operators.coreos.com/operator:v2alpha1/plumbus: ""
-   refs:
+   resources:
```

Comment:
it's actually `refs`, see https://github.com/operator-framework/api/blob/e29d40c2eb3eab6a2789a671886b8966ce8e3c0a/pkg/operators/v1/operator_types.go#L47